### PR TITLE
Bug fix for flight modes names at the chart

### PR DESCRIPTION
**Pull request by demvlad**
This PR resolves issue of wrong additional flight mode names at the charts (The wrong USER3 mode is added to the ANGLE mode for example).
The log file for example:
https://disk.yandex.ru/d/-_Sr2K-atVpA5g
Current BBE version:
WrongMode
This PR:
NormalMode

The caption "Flight mode changed " before flight mode is removed, to get shorter text label.
This is first PR to improve flight modes show in BBExplorer charts.
As i look in BF firmware and BBExplorer source, we have some problem with logging flight modes,
because flight modes description count is more then bit count in 32 bit unsigned value, what is written to log file.
Therefore some modes (ACROTRAINER for example) cannot logged. The new ALTHOLD mode (@ctzsnooze PR [#13816](https://github.com/betaflight/betaflight/pull/13816)) has not true description in BBE too.
I will continue to improve these issues in next PR.

### CODE REVIEW

File: src/flightlog_fields_presenter.js [hunk 1]
```diff
@@ -1337,15 +1337,19 @@ FlightLogFieldPresenter.presentFlags = function (flags, flagNames) {
...
-  flagNames
-) {
+  flagNames)
+{
   let eventState = "";
   let found = false;
-
-  for (let i = 0; i < flagNames.length; i++) {
+  const maxModeNumber = 32;   // int has 32 bit only! We have not to roll bit shift 1<<i for i values grate then 31 !!!
+  let modesCount = flagNames.length
```
Comment (haslinghuis) [inline src/flightlog_fields_presenter.js:1345]:
```suggestion
  let modesCount = flagNames.length;
```

Comment (demvlad) [reply src/flightlog_fields_presenter.js]: Yes, ofcourse, i've missed semicolon.

**Review by haslinghuis — APPROVED** [review]

**Comment by ctzsnooze** [comment]
Thanks @demvlad this already looks much nicer.  Let me know what logs you need.

**Comment by demvlad** [comment]
> Let me know what logs you need.

@ctzsnooze could you send me any log file with short comments - what mode was on/off every time in realy

**Comment by demvlad** [comment]
@ctzsnooze @haslinghuis @nerdCopter 
I've checked. The BF firmware works properly.
It writes long multy-byte value in log and can write flight modes with big number.
But BBExplorer parser tryes to read up 5 byte, but it cannot read last byte properly, because it uses simple int32 variable as result.
Therefore, some flight modes (acrotrainer, autolaunch and other what have enum number gt 32) probably will give wrong mirrors on some other mode and will not show properly.
I suggest to limit size of multi-byte values what BBExplorer parser reads as 32bit till, and will think how to read multy-byte values. properly. The first suggestion will be pushed as PR soon. The second need much time.

**Pull request merged**